Makefile

```diff
@@ -1,4 +1,5 @@
 TEST_OPTIONS=-m unittest discover --start-directory tests --top-level-directory .
+CPU_COUNT=$(shell python -c "from multiprocessing import cpu_count; print(cpu_count())")
```

Comment:
Oo so there is no need to import the print function from __future__?

Interesting :100: